runtime/lua/vim/pack.lua

```diff
@@ -367,13 +382,16 @@ local function new_progress_report(title)
 
   return vim.schedule_wrap(function(kind, percent, fmt, ...)
     local progress = kind == 'end' and 'done' or ('%3d%%'):format(percent)
-    print(('(vim.pack) %s: %s %s'):format(progress, title, fmt:format(...)))
+    local details = (': %s %s'):format(title, fmt:format(...))
+    local chunks = { { '(vim.pack)', 'ModeMsg' }, { ' ' }, { progress, 'WarningMsg' }, { details } }
```

Comment:
Would prefer to use `vim.pack:` (or just `pack:`) as the message prefix. That is the conventional message format. Don't want new features to start copying a different `(foo)` convention.